### ISSUE TITLE
3 High severity CVE's 

### ISSUE THREAD

**Issue by techie2000** [mikenye/docker-picard#37]
```
+--------------------------------+------------------+----------+-------------------------------+--------------------------+-----------------------------------------+
|            LIBRARY             | VULNERABILITY ID | SEVERITY |       INSTALLED VERSION       |      FIXED VERSION       |                  TITLE                  |
+--------------------------------+------------------+----------+-------------------------------+--------------------------+-----------------------------------------+
| libfreetype6                   | CVE-2020-15999   | HIGH     | 2.8.1-2ubuntu2                | 2.8.1-2ubuntu2.1         | freetype: Heap-based buffer             |
|                                |                  |          |                               |                          | overflow due to integer                 |
|                                |                  |          |                               |                          | truncation in Load_SBit_Png             |
|                                |                  |          |                               |                          | -->avd.aquasec.com/nvd/cve-2020-15999   |
+--------------------------------+------------------+----------+-------------------------------+--------------------------+-----------------------------------------+
| libssl1.1                      | CVE-2020-1971    | HIGH     | 1.1.1-1ubuntu2.1~18.04.6      | 1.1.1-1ubuntu2.1~18.04.7 | openssl: EDIPARTYNAME                   |
|                                |                  |          |                               |                          | NULL pointer de-reference               |
|                                |                  |          |                               |                          | -->avd.aquasec.com/nvd/cve-2020-1971    |
+--------------------------------+------------------+----------+-------------------------------+--------------------------+-----------------------------------------+
| openssl                        | CVE-2020-1971    | HIGH     | 1.1.1-1ubuntu2.1~18.04.6      | 1.1.1-1ubuntu2.1~18.04.7 | openssl: EDIPARTYNAME                   |
|                                |                  |          |                               |                          | NULL pointer de-reference               |
|                                |                  |          |                               |                          | -->avd.aquasec.com/nvd/cve-2020-1971    |
+--------------------------------+------------------+----------+-------------------------------+--------------------------+-----------------------------------------+
```

There are hundreds of Medium/Low CVE's, so focussing on the High ones only (for now at least!)

**Comment by mikenye** [comment]
@techie2000 what tool are you using to get the output you pasted above?

**Comment by techie2000** [comment]
@mikenye [trivvy](https://github.com/aquasecurity/trivy)

**Comment by mikenye** [comment]
Thanks

**Comment by mikenye** [comment]
Hi @techie2000, I ended up placing `trivvy` into the image during the build process, using it to determine any packages with vulnerabilities that have a fixed version, and upgrade them.

The change was merged in #38.
